CMake/finders/FindSDL_audiolib.cmake

```diff
@@ -0,0 +1,27 @@
+find_package(PkgConfig)
+pkg_check_modules(PC_SDL_audiolib QUIET SDL_audiolib)
+
+find_path(SDL_audiolib_INCLUDE_DIR aulib.h
+          HINTS ${PC_SDL_audiolib_INCLUDEDIR} ${PC_SDL_audiolib_INCLUDE_DIRS})
+
+find_library(SDL_audiolib_LIBRARY
+             NAMES SDL_audiolib libSDL_audiolib
+             HINTS ${PC_SDL_audiolib_LIBDIR} ${PC_SDL_audiolib_LIBRARY_DIRS})
+
+include(FindPackageHandleStandardArgs)
+find_package_handle_standard_args(SDL_audiolib DEFAULT_MSG
+                                  SDL_audiolib_INCLUDE_DIR SDL_audiolib_LIBRARY)
+
+if(SDL_audiolib_FOUND AND NOT TARGET SDL2::SDL_audiolib)
+  add_library(SDL2::SDL_audiolib UNKNOWN IMPORTED)
```

Comment:
1. `SDL_audiolib` is not a part of SDL2; The name should be `SDL_audiolib::SDL_audiolib`
2. In 3rdParty/SDL_audiolib/CMakeLists.txt, add `add_library(SDL_audiolib::SDL_audiolib ALIAS SDL_audiolib)`
3. Use `SDL_audiolib::SDL_audiolib` as the dependency name throughout.